data/1.0/_index.md

```diff
@@ -44,11 +44,11 @@ garage.save(ferrari);
 
 
 * [Jakarta Data Release Record](https://projects.eclipse.org/projects/ee4j.data/releases/1.0)
-* [Jakarta Data 1.0 Specification Document](./data-1.0.0-b3.pdf) (PDF)
```

Comment:
Looks good. A small request:  Move the list under the "Details" heading as shown in the attached image.

![data](https://github.com/jakartaee/specifications/assets/149188/8b750595-81b0-429d-b1c9-2a314d1d4a5e)